docs/BuildConfiguration.md

```diff
@@ -54,6 +54,9 @@ Sections follow that describe particular settings in more depth.
 |       source                                                             |    `uri`    | The unique URI pointing to the data file.                                                                                                                      | *Required* |                                   |  2.2  |
 | demDefaults                                                              |   `object`  | Default properties for DEM extracts.                                                                                                                           | *Optional* |                                   |  2.3  |
 |    [elevationUnitMultiplier](#demDefaults_elevationUnitMultiplier)       |   `double`  | Specify a multiplier to convert elevation units from source to meters.                                                                                         | *Optional* | `1.0`                             |  2.3  |
+| [digitransitEmissions](#digitransitEmissions)                            |   `object`  | Configure properties for emissions file.                                                                                                                       | *Optional* |                                   |   na  |
+|    carAvgCo2                                                             |  `integer`  | The average CO2 emissions of a car.                                                                                                                            | *Optional* | `170`                             |   na  |
```

Comment:
Would you mind using `CO₂` in comments and documentation, please (using native character `₂`)?

Reply:
I don't mind - made the changes.